src/main/kotlin/uk/co/craigbass/pratura/boundary/AddShippingAddress.kt

```diff
@@ -0,0 +1,17 @@
+package uk.co.craigbass.pratura.boundary
+
+import com.madetech.clean.usecase.SynchronousUseCase
+import uk.co.craigbass.pratura.boundary.AddShippingAddress.Request
+
+interface AddShippingAddress : SynchronousUseCase<Request, Unit> {
+  data class Request(
+    val name: String,
+    val companyName: String?,
+    val addressLine1: String,
+    val addressLine2: String?,
```

Comment:
I suspect the UI would still give the user address line 1, for UK... but think this localisation thing of presenting the user with 1 or 5 rows should be up to whatever UI is slapped on top